lib/src/main/java/lt/neworld/spanner/Spanner.kt

```diff
@@ -142,4 +142,14 @@ class Spanner(text: CharSequence?) : SpannableStringBuilder(text) {
 
         return this
     }
+
+    fun span(startIndex: Int, search: CharSequence, vararg spans: Span): Spanner {
+        if (TextUtils.isEmpty(search)) {
+            return this
+        }
+        val index = TextUtils.indexOf(this, search, startIndex)
+        if (index > -1) setSpans(index, index + search.length, *spans)
```

Comment:
This implementation set span once. But the user could have multiple spans. Your code should be similar to this:

https://github.com/SzasznikaJanos/spanner/blob/3e8c9c52bb74b4b708c1af32f68d5f8fbe7190b1/lib/src/main/java/lt/neworld/spanner/Spanner.kt#L136-L141

Except `lastPos` is user given value. In addition, you should make code DRY. Original method `fun span(search: CharSequence, vararg spans: Span)` should call this method with `startIndex = 0`.